elife/config/etc-init.d-goaws

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
+
+### BEGIN INIT INFO
+# Provides:          goawsdaemon
+# Required-Start:    $local_fs $network $syslog
+# Required-Stop:     $local_fs $network $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: GoAWS
+# Description:       Start/stop daemon for GoAWS.
+### END INIT INFO
+
+NAME="goaws"
+PIDFILE="/var/run/goaws/goaws.pid"
+PATH="/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin"
+APPDIR="/"
+APPBIN="/usr/local/bin/goaws"
+
+# Include functions
+set -e
+. /lib/lsb/init-functions
+
+start() {
+  printf "Starting '$NAME'... "
+  start-stop-daemon --start --background -m --oknodo --pidfile ${PIDFILE} --make-pidfile --chuid root:root --exec ${APPBIN}
+  aws sns create-topic --name=annual-reports --endpoint=http://localhost:4100
+  aws sns create-topic --name=blog-articles --endpoint=http://localhost:4100
+  aws sns create-topic --name=collections --endpoint=http://localhost:4100
+  aws sns create-topic --name=covers --endpoint=http://localhost:4100
+  aws sns create-topic --name=events --endpoint=http://localhost:4100
+  aws sns create-topic --name=interviews --endpoint=http://localhost:4100
+  aws sns create-topic --name=labs-experiments --endpoint=http://localhost:4100
+  aws sns create-topic --name=people --endpoint=http://localhost:4100
+  aws sns create-topic --name=podcast-episodes --endpoint=http://localhost:4100
+  aws sns create-topic --name=press-packages --endpoint=http://localhost:4100
+  aws sns create-topic --name=subjects --endpoint=http://localhost:4100
+  aws sqs create-queue --queue-name journal-cms--queue-local --endpoint=http://localhost:4100
```

Comment:
@giorgiosironi @lsh-0 How would I make a variable in here from pillar here that can be customised by each environment, should I instead break it into a configuration file?